test/CMakeLists.txt

```diff
@@ -3,6 +3,17 @@
 #
 # Rahul, Jeff Ator, Ed Hartnett
 
+# This function builds, links, and runs a test program.
+function(create_test name kind)
+  add_executable(${name}_${kind} ${name}.F90)
+  target_link_libraries(${name}_${kind} PRIVATE bufr_${kind})
+  set_target_properties(${name}_${kind} PROPERTIES COMPILE_FLAGS "${fortran_${kind}_flags}")
```

Comment:
Per below, this should be `PRIVATE bufr_4` (or `PUBLIC bufr:bufr_4`).  In other words, no ${kind} option here, because there's only one build of the library now.

Reply:
OK I will fix.

Reply:
Fixed.